client/src/components/common/loaderComponent/loaderComponent.css

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
I'd suggest use CSS-in-JS to be consistent with rest of the projects. See, `client\src\components\Navbar.js` for an example. But this is not a priority right now so we can make another PR for that.

Reply:
Let's have a discussion over it. I think a separate CSS file is a better approach.